codeformatter/lib/phpbeautifier/CompatInfo/standard_const_array.php

```diff
@@ -260,11 +260,6 @@
     'init' => '5.0.0',
     'name' => 'final',
   ),
-  'FINALLY' =>
-  array (
-    'init' => '5.5.0',
-    'name' => 'finally',
-  ),
```

Comment:
@up9cloud i don't know how phpbeautifier works so can you explain why did you removed FINALLY and how is it related to `support php 5.3 T_GOTO`